troposphere/static/js/components/modals/project/instance_launch/InstanceLaunch.react.js

```diff
@@ -462,6 +463,7 @@ define(function (require) {
             <h4>Projected Resource Usage</h4>
             {this.renderCpuConsumption(identity, size, sizes, instances)}
             {this.renderMemoryConsumption(identity, size, sizes, instances)}
+            <a className="modal-link" href="#" onClick={this.props.onRequest}>Need more resources{String.fromCharCode(63)}</a>
```

Comment:
You can replace String.fromCharCode(63) with question mark in quotes like this:

{"Need more resources?"}